esmvalcore/_main.py

```diff
@@ -368,12 +368,35 @@ def run(self,
             strict (fail if there are any warnings).
         """
         import os
+        import warnings
 
         from ._config import configure_logging, read_config_user_file
         from ._recipe import TASKSEP
         from .cmor.check import CheckLevels
         from .esgf._logon import logon
 
+        # Check validity of optional command line arguments with experimental
+        # API
+        with warnings.catch_warnings():
+            # ignore experimental API warning
+            warnings.simplefilter("ignore")
+            from .experimental.config._config_object import Config
```

Comment:
Avoid name collision with :133

```suggestion
            from .experimental.config._config_object import ExpConfig
```